__tests__/server.test.ts

```diff
@@ -114,7 +115,8 @@ describe('query healthcareProfessionalById', () => {
 
         expect(errors[0].extensions.http).toEqual({status: 404})
         expect(errors.length).toBe(1)
-        expect(errors[0].extensions.code).toBe('NOT_FOUND')
+        expect(errors[0].extensions.code).toBe(ErrorCode.NOT_FOUND)
         expect(errors[0].message).toBe('The healthcare professional does not exist.')
     })
 })
+
```

Comment:
Initially I wanted to add more test for healthcareProfessional, but: 

- addHealthcareProfessional is created when created a Facility I thought this was better to test in Facility tests
- updateHealthcareProfessional doesn't exist yet?
- addHealthcareProfessionalToFacility I have some issues with the adding a Healthcare Professional to a existing Facility using Apollo server interface.